slides/background_python_and_jupyter_notebook.html

```diff
@@ -252,7 +252,7 @@
 
 ## Jupyter Notebook
 
-Please go through the [`Jupyter Notebook jupyter notebook`](../../notebooks/notebooks/introduction_jupyter-notebook.ipynb) now
+Please go to the folder `project/notebooks/` and type in the console [`jupyter notebook introduction_jupyter-notebook.ipynb`](../../notebooks/notebooks/introduction_jupyter-notebook.ipynb) now
```

Comment:
They don't need to go to the console as they should be able to just click on the link.